docs/howto/how-to-channel-based-upgrades.md

```diff
@@ -32,14 +31,15 @@ kind: ClusterExtension
 metadata:
   name: argocd
 spec:
+  namespace: argocd
+  serviceAccount:
+    name: argocd-installer
   source:
     sourceType: Catalog
     catalog:
       packageName: argocd-operator
       channels: [stable] # Automatically upgrade to the latest version found in ‘stable’
-      version: “!1.3.2” # Don’t allow version 1.3.2
-  install:
-    namespace: argocd
-    serviceAccount:
-      name: argocd-installer
+      version: "!=1.3.2" # Don’t allow version 1.3.2
```

Comment:
I panicked because of this doc today:

1. It was using `!` instead of `!=`.
2. After fixing it - it did not work because of `“` instead of `"`

I thought we had a regression in version field validation somewhere.